### PR TITLE
fix: escape chars in filter

### PR DESCRIPTION
**Pull request by lucaCambi77**
see
- https://github.com/dhis2/dhis2-docs/pull/1195
- https://github.com/dhis2/dhis2-docs/pull/1205

### CODE REVIEW

File: src/developer/web-api/tracker.md [hunk 1]
```diff
@@ -1258,7 +1258,7 @@ The endpoint returns a list of tracked entities that match the request parameter
...
+|`filter`|`String`|Comma separated values of attribute filters|Narrows response to TEIs matching given filters. A filter is a colon separated property or attribute UID with optional operator and value pairs. Example: `filter=H9IlTX2X6SL:sw:A` with operator starts with `sw` followed by a value. Special characters like `+` need to be percent-encoded so `%2B` instead of `+`. Characters such as `:` (colon) or `,` (comma), as part of the filter value, need to be escaped by `/` (slash). Likewise, `/` needs to be escaped. Multiple operator/value pairs for the same property/attribute like `filter=AuPLng5hLbE:gt:438901703:lt:448901704` are allowed. Repeating the same attribute UID is not allowed. User needs access to the attribute to filter on it.|
```
Comment (teleivo) [inline src/developer/web-api/tracker.md:1261]: We are moving the API docs of parameters to OpenAPI spec 😄 The parameters table will be removed once we have documented all parameters. Please add this sentence to https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md#parametertrackedentityrequestparamsfilter (and all related controllers that have this parameter).

File: src/developer/web-api/tracker.md [hunk 2]
```diff
@@ -1345,6 +1345,12 @@ item:
         &filter=lw1SqmMlnfh:GT:150
         &filter=lw1SqmMlnfh:LT:190
 
+A query with characters in the filter value that need escaping:
```
Comment (teleivo) [inline src/developer/web-api/tracker.md]: It helps me (and maybe other users) to see the actual value that is being sent and how the value is interpreted. I assume the value will be interpreted as `:,/`, right?

File: src/developer/web-api/tracker.md (hunk 1 above)
Comment (lucaCambi77) [reply src/developer/web-api/tracker.md:1261]: This is 2.40, I only see that in master?

Comment (teleivo) [reply src/developer/web-api/tracker.md:1261]: Ah, sorry, you are right! What I said affects the docs on master (2.41) :)

File: src/developer/web-api/tracker.md (hunk 2 above)
Comment (lucaCambi77) [reply src/developer/web-api/tracker.md]: yes

File: src/developer/web-api/tracker.md (hunk 1 above)
Comment (lucaCambi77) [reply src/developer/web-api/tracker.md:1261]: ok i'll do a PR for master then

File: src/developer/web-api/tracker.md (hunk 2 above)
Comment (teleivo) [reply src/developer/web-api/tracker.md]: Can you please add that the value will be interpreted as `:,/`

**Review by teleivo — APPROVED** [review]

**Comment by teleivo** [comment]
Please remember to add

> Characters such as `:` (colon) or `,` (comma), as part of the filter value, need to be escaped by `/` (slash). Likewise, `/` needs to be escaped.

to our OpenAPI spec in https://github.com/dhis2/dhis2-core/tree/master/dhis-2/dhis-web-api/src/main/resources/openapi. Thank you 😄

**Pull request merged**